Frontend/src/app/services/assets.service.ts

```diff
@@ -9,9 +9,17 @@ import { baseUrl } from '../../global';
 export class AssetsService {
 	constructor(private readonly httpClient: HttpClient) {}
 
-	public getAsset(fileName: string): Observable<string> {
+	public getAssetForRestaurant(fileName: string): Observable<string> {
 		return this.httpClient
-			.get(`${baseUrl}/assets?assetName=${fileName}`, {
+			.get(`${baseUrl}/assets/restaurant?assetName=${fileName}`, {
```

Comment:
You should make `assets` a variable, and make a more generic function that takes a subpath and a filename that way.
```suggestion
			public getAssetForRestaurant(subpath: string, fileName: string): Observable<string>
```